x/tools/fetch/main.go

```diff
@@ -71,10 +72,14 @@ func overrideAddress(original string, newHost string, newPort string) (string, e
 
 func main() {
 	verboseFlag := flag.Bool("v", false, "Enable debug output")
+
 	tlsKeyLogFlag := flag.String("tls-key-log", "", "Filename to write the TLS key log to allow for decryption on Wireshark")
-	protoFlag := flag.String("proto", "h1", "HTTP version to use (h1, h2, h3)")
-	transportFlag := flag.String("transport", "", "Transport config")
+	echConfigFlag := flag.String("ech-config", "", "Base64-encoded ECH config")
```

Comment:
Do we have any documentation of this config?

Reply:
If we don't have docs elsewhere it would be nice to include something like your verification example showing how the base64 string was generated.

Reply:
It's documented in the RFC: https://www.ietf.org/archive/id/draft-ietf-tls-esni-25.html#name-encrypted-clienthello-confi

To test, simply run dig on a domain that has it